.docker/docker-compose.yaml

```diff
@@ -0,0 +1,44 @@
+version: "3"
+services: 
+  isaac-sim:
```

Comment:
What is the purpose of starting 2 containers (`isaac-sim:2022.1.1-ubuntu20.04` + `orbit:2022.1.1-ubuntu20.04`)? 
Wouldn't `orbit:2022.1.1-ubuntu20.04` already include everything FROM its base image `isaac-sim:2022.1.1-ubuntu20.04`?